box-share-sdk/src/main/java/com/box/androidsdk/share/vm/SharedLinkVM.java

```diff
@@ -46,16 +44,7 @@ public void disableSharedLink(BoxCollaborationItem item) {
         return mShareLinkedItem;
     }
 
-    public void setActiveRadioButtons(HashSet<BoxSharedLink.Access> activeRadioButtons) {
-        this.mActiveRadioButtons = activeRadioButtons;
-    }
-
-    public HashSet<BoxSharedLink.Access> getActiveRadioButtons() {
-        return mActiveRadioButtons;
-    }
-
-
-    public HashSet<BoxSharedLink.Access> generateActiveButtons() { //move to VM in the future
+    public HashSet<BoxSharedLink.Access> getActiveRadioButtons() { //move to VM in the future
```

Comment:
is the comment still relevant? this is already in the VM, which VM is the comment referring to?

Reply:
It's not. I made a note for this since this was in Fragment before. I will remove the comment.